sparkbox/designsystemssurvey
# Basic auth via marshmallows

## Steps to test

1. Run the server as usual with `NODE_ENV=production` (`NODE_ENV=production npm run server`
2. http://localhost:3000
**Expected:** No prompt for credentials

1. Run the server with `USER` and `PASS` (`NODE_ENV=production USER=test PASS=test npm run server`)
2. http://localhost:3000
**Expected:** Prompted for basic credentials

## Steps for production
`USER` and `PASS` are configured for production. During launch, remove these to disable basic auth
